packages/tools/src/generateSpellsData.ts

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable */
+import fs from 'fs-extra';
+import path from 'path';
+
+import spellDBInfo from '../../../../pvp-tooltips-addon/spellDataParsed.json';
+import spellEffects from '../../shared/src/data/spellEffects.json';
+import spellTags from '../../shared/src/data/spells.json';
+
+//spellTags
+// "66": {
+//   "type": "buffs_offensive"
+// },
+// "99": {
+//   "type": "cc"
+// },
+// "118": {
+//   "type": "cc"
+// },
+// "122": {
+//   "type": "roots"
+// },
+// "130": {
+//   "type": "buffs_other"
+// },
+
+// spellDB
+// type SpellInfo = {
+//   spellId: number;
+//   name: string;
+//   pvpModifiers: PvPModifier[];
+//   /**
+//    * Spells that directly (1-hop) trigger this spell; DB entry
+//    */
+//   triggeredBy: number[];
+//   /**
+//    * Spells that directly (1-hop) affect this spell; DB entry
+//    */
+//   affectedBy: number[];
+//   /**
+//    * All spells that this spell triggers; computed
+//    */
+//   triggerChain: number[];
+//   /**
+//    * All spells that affect this spell; computed
+//    */
+//   azeritePowerId: string;
+//   affectedByChain: number[];
+//   pvpModifiersFromTriggers: PvPModifier[];
+//   pvpModifiersFromAffectors: PvPModifier[];
+
+//   duration: number | null;
+//   cooldown: number | null;
+//   charges: number | null;
+//   chargesCooldown: number | null;
+//   gcd: number | null;
+// };
+
+// spellEffects
+// "66": {
+//   "spellId": "66",
+//   "name": "Invisibility",
+//   "cooldownSeconds": 300,
+//   "charges": {},
+//   "durationSeconds": 3
+// },
+// "88625": {
+//   "spellId": "88625",
+//   "name": "Holy Word: Chastise",
+//   "cooldownSeconds": 60,
+//   "charges": {},
+//   "durationSeconds": 4
+// },
+//
+// "charges": {
+//   "charges": 1,
+//   "chargeCooldownSeconds": 30
+// },
+// "durationSeconds": 15
+
+async function main() {
+  console.log('Comparing SL->DF effects data');
+  Object.keys(spellEffects).forEach((k) => {
```

Comment:
this assumes spellEffects object already has an exhaustive list of spell IDs? is that intended?